ravenframework/contrib/PythonFMU/pythonfmu.egg-info/SOURCES.txt

```diff
@@ -0,0 +1,86 @@
+LICENSE
```

Comment:
Specifically, the entire pythonfmu.egg-info directory should not be committed.  (I forgot to put this in the .gitignore, but the .gitignore has since been updated)

Reply:
@joshua-cogliati-inl Thanks for the comments. I have modified the committed content.

Reply:
Thanks for fixing :)